db/schema.rb

```diff
@@ -49,6 +49,7 @@
     t.datetime "updated_at"
     t.datetime "applied_at"
     t.integer  "updater_id"
+    t.text     "state",                       default: "draft", null: false
```

Comment:
> I decided to go with state because a few of the controllers/helpers depend on a state method/attribute being available

Ah, you're right. I thought the calls to `state` were all changed in favor of the aasm predicates. But in that case, it would even count as another argument against adding aasm as adding it doesn't fully hide away the underlying state implementation or state-holding data attributes.